addons/mass_mailing/static/src/scss/mass_mailing.scss

```diff
@@ -14,7 +14,18 @@
         }
     }
 }
-
+.o_form_view {
+    // Small adjustment on Widget(sms x emojis) added to Text field.
+    .o_mail_emojis_mass_mailing ~ .o_mail_add_emoji{
+        margin-top: -55px;
+    }
```

Comment:
Is there not a cleaner way to do this?
Negative margin looks like a dirty fix.

Reply:
Yes I agree :) 
But this workaround was used in the [widget code](https://github.com/odoo/odoo/blob/00404541e05efddf491e6f41718595f2313d7b98/addons/mail/static/src/scss/emojis.scss#L39) itself (the emojis widget is displayed outside the text area)

Reply:
While you're at it, maybe you could change the default implementation with position-relative class and add a default `bottom: 40px` that you override to 55px for the sms widget?

To test but it would be cleaner.